stdlib/3/asyncio/tasks.pyi

```diff
@@ -93,9 +93,9 @@ def wait_for(fut: _FutureT[_T], timeout: Optional[float],
 
 class Task(Future[_T], Generic[_T]):
     @classmethod
-    def current_task(cls, loop: Optional[AbstractEventLoop] = ...) -> Task[Any]: ...
+    def current_task(cls, loop: Optional[AbstractEventLoop] = ...) -> Task[_T]: ...
```

Comment:
I don't think it makes sense to change it to `_T` on the methods without `_T` parameters. `_T` isn't bound outside a class, or on the class methods. Also, all_tasks and current_task aren't guaranteed to return any specific type

Reply:
@CraftSpider Thanks for the review, I am just curious how did you find out that `all_tasks` and `current_task` aren't guaranteed to return any specific type. I am a beginner.

Reply:
@CraftSpider Thanks for the review, I am just curious how did you find out that all_tasks and current_task aren't guaranteed to return any specific type. I am a beginner.

Reply:
They do return `Task`, by 'any type' I mean `Task[Any]` not `Task[_T]`. They return either the current task or all unfinished tasks. Personally, I've used asyncio and just remembered; but for the actual explanation, https://docs.python.org/3/library/asyncio-task.html#introspection is the documentation for them.

Reply:
They do return `Task`, by 'any type' I mean `Task[Any]` not `Task[_T]`. They return either the current task or all unfinished tasks. Personally, I've used asyncio and just remembered; but for the actual explanation, https://docs.python.org/3/library/asyncio-task.html#introspection is the documentation for them.

Reply:
@CraftSpider Yeah should have checked the docs, `current_task` can also return `None`, so put an `Optional`, thanks.